llvm/lib/Transforms/Instrumentation/AddressSanitizer.cpp

```diff
@@ -1234,7 +1245,91 @@ static bool isUnsupportedAMDGPUAddrspace(Value *Addr) {
   return false;
 }
 
+static bool isUnsupportedSPIRAddrspace(Value *Addr, Function *Func) {
```

Comment:
`Func` argument seems to be unused

Reply:
I changed the algorithm, it's useful now